v2/emailpassword/common-customizations/sessions/session-verification-in-api/get-session.mdx

```diff
@@ -0,0 +1,96 @@
+---
+id: get-session
+title: Using the getSession function
+hide_title: true
+---
+
+
+<!-- COPY DOCS -->
+<!-- ./session/common-customizations/sessions/session-verification-in-api/get-session.mdx -->
+
+import BackendSDKTabs from "/src/components/tabs/BackendSDKTabs"
+import TabItem from '@theme/TabItem';
+
+
+// add try catch, add all possible errors and in comments decribe how to handle them, also add the other two ways of doing session verification.
+
+# Session Verification using `getSession`
+
+If you need to verify a session in your API, you should be using the [verifySession](./verify-session) middleware whenever possible, but, if you need session verification in the middle of your API logic, then use `getSession` as shown below.
```

Comment:
- Make this an important tool tip
- Change the message to "If you can use `verifySession` instead, please use that since it is simpler to use."